potentials/Cu-PBE-core-rep.ace

```diff
@@ -0,0 +1,8980 @@
+nelements=1
```

Comment:
It will be the 45th largest file in potentials dir, increasing dir size by about 1%, so I don't think we should exclude it.  There is some benefit to letting people take a look at it without having to build the package.